src/lib/ui/sync-factory.js

```diff
@@ -26,6 +26,7 @@ module.exports = function(T, t, api) {
     // selfies
     currentpage: '1',
     events: {
+      'pane.activated': M('uiSync'),
```

Comment:
The `M()` here is redundant, since all it does is replacing the `<T>` placeholder. I think you can omit it.